docker/loom-delta-test/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+tag=$1
+
+if [ -z $tag ]; then
+    echo -e "\nYou must provide a tag"
+    echo -e "\nUsage: bash build_docker.sh TAG\n"
+    exit 1
+fi
+
+dockerName=loom-delta-test
+
+docker build -t quay.io/humancellatlas/${dockerName}:${tag} .
```

Comment:
This is 100% fine for now, but as read more about the ticket description, if the main audience for this PR is non-HCA people, it might be a good idea to have its standalone quay docker repo than under HCA in the future, to minimize the potential impacts from HCA.